06 piškvorky/piskvorky1D.py

```diff
@@ -0,0 +1,77 @@
+from random import randrange
+import time
+
+# TICK-TACK-TOE
+
+
+def score(area):  # fce get list(area) and return result in accordance with area
+
+    # draw
+    if '-' not in area:
+        print(' "!" ')
+        quit()
+    # players won
+    elif (3 * player_symbol) in area:
+        print(' "X" ')
+        quit()
+    # pc won
+    elif (3 * pc_symbol) in area:
+        print('"O"')
```

Comment:
Ty printuješ len ten znak, ale skôr by to chcelo niečo jednoznačnejšie ako : Hráč vyhrál/PC vyhrál.

Reply:
jooo, to jsem tam měla :-D ale ze zadání jsem pochopila, že to má vypsat znak, tak jsem to upravila 
9. Napiš funkci vyhodnot, která dostane řetězec s herním polem 1-D piškvorek a vrátí jednoznakový řetězec
podle stavu hry:
"x" – Vyhrál hráč s křížky (pole obsahuje xxx)
atd. 

Jinak určitě souhlasím :)

Reply:
Ooo tak, no to je blbé zadanie teda :D